fastlane/Fastfile

```
@@ -0,0 +1,61 @@
+      task: "signingReport"
+      keystore_path: ENV['KEYSTORE_PATH'],
+      keystore_password: ENV['KEYSTORE_PASSWORD'],
+      key_alias: ENV['KEY_ALIAS'],
+      key_password: ENV['KEY_PASSWORD']
+    )
+
+    # Deploy to Google Play for Production
+    supply(
+      track: "internal",
+      aab: "app/build/outputs/bundle/#{options[:variant]}Release/app-#{options[:variant]}-release.aab"
+      json_key: "./google_play_service_account.json"
```

rubocop (1.69.1)

[fatal] unexpected token tIDENTIFIER
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)